### DISCUSSION TITLE
where to enter item dimensions and weight

### DISCUSSION THREAD

**Discussion by bindalf** [etsy/open-api#1222] [Q&A]
Where to enter these values in the Listing?
![image](https://github.com/etsy/open-api/assets/123418968/b0fe76d8-6fb1-4c8e-89df-bb0a39b25a4f)

**Comment by liam-strong** [comment]
You simply pass through the data in the array being sent to the API.
```python
width = 200
height = 200
data = {
"item_width": width,
"item_height": height,
}
```

**Reply by bindalf** [reply]
i mean where on the Etsy Website can a shopowner enter these values for a listing

**Reply by liam-strong** [reply]
You would need to login to the listing manager, select the listing you want to edit and scroll down the text input fields are half way down the page. 
![image](https://github.com/etsy/open-api/assets/21028906/fcab0b74-5cb5-4787-bf63-1444ca46cb13)

**Reply by bindalf** [reply]
i dont have this, maybe not available in germany?

![image](https://github.com/etsy/open-api/assets/123418968/5d6595e0-651d-46de-97cc-4b27c3308aa1)

**Reply by liam-strong** [reply]
It may be beneficial to directly reach out to Etsy to inquire about the specifics.
